frontend/app-development/features/appSettings/components/TabsContent/Tabs/MaskinportenTab/AnsattportenLogin/AnsattportenLogin.test.tsx

```
@@ -0,0 +1,65 @@
+
+describe('getRedirectUrl', () => {
+  it('should build and return correct redirect url', () => {
+    mockWindowLocationHref();
+    const result = getRedirectUrl();
+    expect(result).toBe(`/path/to/page?${settingsPageQueryParamKey}=maskinporten`);
+  });
+});
+
+function mockWindowLocationHref(): jest.Mock {
+  const hrefMock = jest.fn();
+  delete window.location;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)